rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/JobRequest.groovy

```diff
@@ -27,6 +27,7 @@ import groovy.transform.Immutable
 class JobRequest {
   List<String> tokenizedCommand
   List<String> maskedPackerParameters = []
+  String jobId = ""
```

Comment:
Please add a comment explaining the relationship between this `jobId` and `BakeRequest.requestId`.